packages/api/src/ApiClient.ts

```diff
@@ -227,6 +228,16 @@ export class ApiClient {
 		return new HelixChannelPointsApi(this);
 	}
 
+	/**
+	 * The Helix charities API methods.
+	 *
+	 * @beta
+	 */
+	@CachedGetter()
+	get charities(): HelixCharityApi {
```

Comment:
As "charity" is more widely used as a singular word, I'd name this property in singular as well.